rainbowsaddle/__init__.py

```diff
@@ -103,7 +103,8 @@ def wait_pid(self, pid):
         """
         try:
             os.waitpid(pid, 0)
-        except OSError, err:
+        except OSError:
```

Comment:
Can you please change this to `except OSError as err`? It will avoid the need to call `sys.exc_info()`